sway/tree/output.c

```diff
@@ -92,10 +97,6 @@ void output_enable(struct sway_output *output, struct output_config *oc) {
 		ipc_event_workspace(NULL, ws, "init");
 	}
 
-	size_t len = sizeof(output->layers) / sizeof(output->layers[0]);
-	for (size_t i = 0; i < len; ++i) {
-		wl_list_init(&output->layers[i]);
-	}
 	wl_signal_init(&output->events.destroy);
```

Comment:
We should call `wl_signal_init` early too.

Reply:
Done.